src/engine/rust.rs

```diff
@@ -586,6 +754,7 @@ pub struct PayloadWrapper {
 #[derive(Debug, Serialize)]
 pub struct Payload {
 	pub bitworkc: String,
+	pub bitworkr: Option<String>,
```

Comment:
Is this part necessary?

Reply:
The intention is trying to keep sync and consistent with atomicals-js. Maybe not necessary.
I will submit a new PR soon for your review. Thanks for your advice.